src/applications/personalization/connected-accounts/sass/connected-acct.scss

```diff
@@ -64,6 +64,13 @@
          }
       }
    }
+
+   .va-connected-acct-row-details-toggle {
```

Comment:
Are you trying to make this button look like a link? We have a class called `va-button-link` that will do that for you.

Otherwise, you should replace the color here with the `$color-link-default` variable.

Reply:
yeah, i was trying to keep the visual presentation similar to what it was before. i'll see which one makes the most sense? i am also to input on what the button should look like now that it is a button